openapi3/example_validation_test.go

```diff
@@ -26,7 +26,7 @@ func TestExamplesSchemaValidation(t *testing.T) {
             param1example:
               value: abcd
    `,
-			errContains: "invalid paths: param1example",
+			errContains: "param1example",
```

Comment:
I'd like to see the whole following be matched:
> invalid paths: invalid path /user: invalid operation POST: param1example: Field must be set to integer or not be present

Reply:
Done. I've updated it.